omero_marshal/decode/decoders/transform.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Glencoe Software, Inc. All rights reserved.
+#
+# This software is distributed under the terms described by the LICENCE file
+# you can find at the root of the distribution bundle.
+# If the file is missing please request a copy by contacting
+# jason@glencoesoftware.com.
+#
+
+from ... import SCHEMA_VERSION
+from .annotation import Decoder
+try:
```

Comment:
Like we have done in the rectangle decoder, let's make explicit comments why we are doing this.